src/content/overview/fleet-management/cluster-management/cluster-concepts/releases/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Workload cluster releases
+description: Details on the workload cluster release offered by Giant Swarm and ways to look up even more details.
+last_review_date: 2025-07-01
+weight: 30
+menu:
+  principal:
+    parent: overview-fleetmanagement-clustermanagement-concepts
+    identifier: overview-fleetmanagement-clustermanagement-concepts-releases
+user_questions:
+  - What is a workload cluster release?
+  - Where can I find details about workload cluster releases?
+  - What is a major workload cluster release?
+  - What is a minor workload cluster release?
+  - What is a patch workload cluster release?
+  - How long are workload cluster releases supported?
+  - What does it mean when a workload cluster release is deprecated?
+  - How soon does Giant Swarm provide new Kubernetes versions?
+  - How are workload cluster releases announced?
+  - What is a pre-release?
+  - What is an alpha release?
+  - What is a beta release?
+  - What is a release candidate?
+aliases:
+  - /platform-overview/cluster-management/releases
+  - /reference/workload-cluster-release-versions/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-tenet
+---
```

Comment:
We can take over the pretty good source article `src/content/vintage/platform-overview/cluster-management/releases/index.md` but remove everything from `Inspecting workload cluster releases`

Reply:
yeah I have opened a new PR for that